src/callbacks/wandb_callbacks.py

```diff
@@ -67,6 +106,10 @@ def __init__(self, ckpt_dir: str = "checkpoints/", upload_best_only: bool = Fals
         self.ckpt_dir = ckpt_dir
         self.upload_best_only = upload_best_only
 
+    @rank_zero_only
+    def on_keyboard_interrupt(self, trainer, pl_module):
+        self.on_train_end(trainer, pl_module)
+
```

Comment:
I added this hook so ckpts will still be uploaded even when run is interrupted with `ctrl + c`